lib/smart_proxy_dynflow/helpers.rb

```diff
@@ -23,6 +23,12 @@ def task_status(task_id)
         status 404
         {}
       end
+
+      def tasks_count(state)
+        filter = state ? { :filters => { :state => [state] } } : {}
+        tasks = world.persistence.find_execution_plans(filter)
+        { :count => tasks.count, :state => state || 'all' }
```

Comment:
I've updated the code before merging, so that we can proceed with PRs on foreman-tasks side